src/js/components/Select/__tests__/__snapshots__/Select-multiple-test.js.snap

```diff
@@ -2194,8 +2194,9 @@ exports[`Select Controlled multiple onChange with valueKey string 3`] = `
 }"
 `;
 
-exports[`Select Controlled multiple onChange without valueKey 1`] = `
```

Comment:
Note: These snapshots still exist, but have been displaced in order with the additional tests.